src/main/java/application/ConversationUsers.java

```diff
@@ -0,0 +1,91 @@
+package application;
+
+import entities.Conversation;
+import entities.User;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class ConversationUsers extends ConversationInteractor {
+    private final Conversation conversation;
+
+    /**
+     * Construct a new ConversationUsers object to interact with a Conversation.
+     *
+     * @param conversation Conversation to interact with
+     */
+    public ConversationUsers(Conversation conversation) {
+        this.conversation = conversation;
+    }
+
+    /**
+     * A request to be carried out by ConversationUsers.
+     */
+    public class ConversationUsersRequest extends RequestModel{
+        private OutputBoundary respondTo;
+        private String action;
+        private User user;
+
+        /**
+         * Fills in this RequestModel's instance attributes.
+         */
+        public void fillRequest(OutputBoundary respondTo, String action, User user) {
+            this.respondTo = respondTo;
+            this.action = action;
+            this.user = user;
+        }
+    }
+
+    /**
+     * @return  A request model to be filled in by caller
+     */
+    public ConversationUsersRequest getRequestModel(){
+        return new ConversationUsersRequest();
+    }
+
+    /**
+     * Accept a given (RequestModel) request.
+     *
+     * @param request1   a request stored as a RequestModel
+     */
+    @Override
+    public void request(RequestModel request1){
+        ConversationUsersRequest request = (ConversationUsersRequest) request1;
+        String action = request.action;
+        if (Objects.equals(action, "addUser")) {
+            this.addUser(request.user);
+        } else if (Objects.equals(action, "removeUser")) {
+            this.removeUser(request.user);
+        } else if (Objects.equals(action, "getUserlist")){
```

Comment:
```suggestion
        } else if (Objects.equals(action, "getUserList")){
```

Last thing! haha

I am just concerned about controller stuff. Not really an issue, but may as well be consistent.